src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,8 @@
+interface NavBarProps {
+  listNavItems: string[];
```

Comment:
`items` is enough
`string[]` is not right, nav items should have destination for navigation on it